regexes/client/browsers.yml

```diff
@@ -515,6 +515,27 @@
   name: 'Baidu Spark'
   version: '$1'
 
+# Yandex Browser (alpha)
+- regex: 'YaBrowser(?:/(\d+[\.\d]*)) \(alpha\)?'
+  name: 'Yandex Browser (alpha)'
+  version: '$1'
+  engine:
+    default: 'Blink'
+
+# Yandex Browser (beta)
+- regex: 'YaBrowser(?:/(\d+[\.\d]*)) \(beta\)?'
```

Comment:
Is the last `?` on the right position?

Reply:
Yes, is based on the regex for Yandex Browser.